subsys/bluetooth/controller/hci_driver.c

```diff
@@ -40,7 +41,10 @@ void blectlr_assertion_handler(const char *const file, const u32_t line)
 
 static int cmd_handle(struct net_buf *cmd)
 {
-	if (hci_cmd_put(cmd->data)) {
+	int errcode;
```

Comment:
errcode should be int32_t

Reply:
This was pointed by @thomasstenersen , and hence I used the same datatype for the errorcode  as that that of the return type of the function.

Is there any guideline on this? Which datatype sets to use in the NCS? 
For ex: int (stdint) vs s32_t (zephyr) vs (int32_t) Nordic?